test/AssemblyContext.cpp

```diff
@@ -82,7 +83,8 @@ INSTANTIATE_TEST_CASE_P(
       // A very long string, encoded after an initial word.
       // SPIR-V limits strings to 65535 characters.
       {std::string(65535, 'a'), {1}},
-    }));
+    }),
+    &DefaultParameterPrinter<EncodeStringCase>);
```

Comment:
Turns out the variable-arguments list may be empty.  This compiles cleanly with `--pedantic`:

```
#define foo(a, ...) a
int main() {
  return foo(0,);
}
```